src/components/ConnectToXeroButton/index.native.tsx

```diff
@@ -0,0 +1,101 @@
+import React, {useRef, useState} from 'react';
```

Comment:
I understand that QBO and Xero almost has the same setup and there could be other integrations with different setup, but I can see 90% of the code is same between both the buttons. Should we DRY the code?

We could have button implementation based on integration type? 

cc - @mountiny

Reply:
Totally agree we could DRY this code, I was just following design doc at that point. The real question is will there be integrations with different setup(I suppose they also will be similar)